### PR TITLE
2539 template

### PR DESCRIPTION
**Pull request by sanjuroj**
Add cell validation, fix tests.

### CODE REVIEW

File: indicators/views/bulk_indicator_import_views.py [hunk 1]
```diff
@@ -121,45 +130,74 @@ def get(self, request, *args, **kwargs):
...
+        validation_map[VALIDATION_KEY_DIR_CHANGE] = dir_change_validation
 
         target_freq_options = list(self.reverse_field_name_map['target_frequency'].keys())
         target_freq_validation = DataValidation(
             type="list", formula1=f'"{",".join(target_freq_options)}"', allow_blank=False)
-        validation_map["Target frequency"] = target_freq_validation
+        validation_map[VALIDATION_KEY_TARGET_FREQ] = target_freq_validation
+
+        # Can't do sector the same way as the others because there are too many.  There is a 256 char limit
+        # on putting options directly into the validation config.  Inst4ad, the options will be referenced
```
Comment (andrethomas6) [inline indicators/views/bulk_indicator_import_views.py:167]: Theres a typo for _Instead_

File: indicators/views/bulk_indicator_import_views.py [hunk 2]
```diff
@@ -58,6 +66,7 @@
 ERROR_NO_NEW_INDICATORS = 101
 ERROR_UNDETERMINED_LEVEL = 102
 ERROR_TEMPLATE_NOT_FOUND = 103
+ERROR_MISMATCHED_TIERS = 104
 ERROR_INDICATOR_DATA_NOT_FOUND = 104
```
Comment (andrethomas6) [inline indicators/views/bulk_indicator_import_views.py:70]: Theres two 104 codes.

**Review by andrethomas6 — APPROVED** [review]

File: indicators/views/bulk_indicator_import_views.py (hunk 1 above)
Comment (sanjuroj) [reply indicators/views/bulk_indicator_import_views.py]: thx

File: indicators/views/bulk_indicator_import_views.py (hunk 2 above)
Comment (sanjuroj) [reply indicators/views/bulk_indicator_import_views.py]: so picky :-)

**Pull request merged**